app/lib/personal_for_you.rb

```diff
@@ -49,19 +49,18 @@ def acct_relay_users
 
   # Get Mammoth user details
   # Includes any settings/preferences/configurations for feeds
+  # Not caching user. If it becomes an issue cache it at the source. AcctRelay
   def user(acct)
-    Rails.cache.fetch(key(acct), expires_in: 60.seconds) do
-      response = HTTP.headers({ Authorization: ACCOUNT_RELAY_AUTH, 'Content-Type': 'application/json' }).get(
-        "https://#{ACCOUNT_RELAY_HOST}/api/v1/foryou/users/#{acct}"
-      )
-      JSON.parse(response.body, symbolize_names: true)
-    end
+    response = HTTP.headers({ Authorization: ACCOUNT_RELAY_AUTH, 'Content-Type': 'application/json' }).get(
```

Comment:
no more caching on Moth.Social. Source of truth is on AcctRelay. This will eventually bite us. If this call really becomes excessive we can cache on acctRelay